examples/endian_swapper/tests/test_endian_swapper.py

```diff
@@ -202,4 +202,5 @@ def wavedrom_test(dut):
         yield tb.csr.read(0)
         yield RisingEdge(dut.clk)
         yield RisingEdge(dut.clk)
-        dut._log.info(waves.dumpj())
+        dut._log.info(waves.dumpj(header = {'text':'WaveDrom example', 'tick':0}))
+        waves.write('wavedrom.json', header = {'tick':0}, config = {'hscale':3})
```

Comment:
Would you mind extending the test to check if the wavedrom.json file is actually written, or even better, that it contains the output you're expecting it to have?